### DISCUSSION TITLE
[DRM] consider support for seatd?

### DISCUSSION THREAD

**Discussion by Ziqi-Yang** [raysan5/raylib#3797] [Features]
Running raylib applications in DRM mode and getting  input requires permission (such as `root` permission for `input` group access).  
[seatd](https://sr.ht/~kennylevinsen/seatd/) provides a way to read access to shared devices (e.g. input) without requiring root privilege. It is used primarily for [wlroots compositors](https://docs.voidlinux.org/config/graphical-session/wayland.html#standalone-compositors), see [voidlinux docs](https://docs.voidlinux.org/config/session-management.html).

**Comment by gen2brain** [comment]
You don't need root privileges, you need a user that is a member of, usually `input` group. You just prepare users for DRM, it is not like that is what most users use, if they use it, someone already configured it like that.

**Comment by Ziqi-Yang** [comment]
Actually I'm developing a greeter(front end) for [greetd](https://sr.ht/~kennylevinsen/greetd/)(linux display manager daemon).  Due to security concern the executor user (called `greeter`) isn't inside the `input` group. Also `multi-seat capabilities` (i.e. one computer, many keyboard etc. ) should be noted.
